components/locale/Switcher.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+  const { locale, locales } = useI18n()
+  // generates URLs to switch between locales
+  const switchLocalePath = useSwitchLocalePath()
+  const availableLocales = computed(() => {
+    // returns a list of locales, exluding current locale
+    return locales.value.filter((i) => i.code !== locale.value)
+  })
+  const currentLocale = computed(() => {
+    return locales.value.filter((i) => i.code === locale.value)
+  })
+</script>
+
+<template>
+  <VMenu transition="scale-transition">
+    <!-- activator btn -->
+    <template #activator="{ props }">
+      <VBtn
+        v-bind="props"
+        rounded="lg"
+        prepend-icon="mdi-translate"
+        append-icon="mdi-chevron-down"
+        variant="text"
+      >
+        <!-- dependent on current locale-->
+        {{ currentLocale[0].code.toUpperCase() }}
```

Comment:
After changing to `.find` in the computed property, remove `[0]`.

Reply:
Should be fixed